src/test/java/de/unistuttgart/iste/gits/content_service/ContentServiceApiTests.java

```diff
@@ -20,11 +21,12 @@
 @SpringBootTest(classes = ContentServiceApplication.class)
 @AutoConfigureGraphQlTester
 @ActiveProfiles("test")
+@Testcontainers
```

Comment:
Should we put that in the "GraphQlApi" annotation maybe? No need to do it now, but in the future